docusaurus/docs/manage/02-router/04-update/01-cli.md

```diff
@@ -0,0 +1,99 @@
+---
+id: router-update
+title: Cli
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import styles from '../styles.module.css';
+
+### Overview
+You can create, list, update and delete routers. In this article
+we are highlighting Cli commands and various options that can be 
+set / updated.
+
+:::info Important Note
+Routers or their identities can be referenced by `@router_name` or `#attribute` in various policies like service policy, edge router policy, etc. Using group `#attribute` is strongly recommended.
+
+&nbsp;
+
+Let's consider an Autoscaling Group scenario, where routers would be created or deleted as the scale-out or scale-in events occur respectively. If router names were referenced in such deployment, then all policies would need to be updated upon the scale-out event with `@router_name`. To keep the complexity of this deployment to minimum, it just makes sense to use #attribute, where no other updates would be needed.
+:::
+
+
+### Create Router
+The router creation command is described with the minimum required options to create the type shown. For the more detail options list, please refer to the [Flags Section](#flags). 
+
+<Tabs groupId="routerType">
+<TabItem value="Private" label="Private Router with Edge" attributes={{className: styles.green}}>
+
+```bash
+ziti edge create edge-router $ROUTER_NAME \
+                            --jwt-output-file $ROUTER_NAME.jwt
+```
+
+</TabItem>
+<TabItem value="Gateway" label="Private Router with Edge and Tunneler" attributes={{className: styles.green}}>
+
+:::info Notes
+`--no-travesal` flag is not required, but keep in mind that private routers are stub routers and setting it to true disables transitive routing through it.
+In other words, only connections destined for this router will be routed to it by the samrt routing algorithm. `--tunneler-enabled or just -t` flag indicates the tunnel mode.
+:::
+
+```bash
+ziti edge create edge-router $ROUTER_NAME \
+                            --jwt-output-file $ROUTER_NAME.jwt \
+                            --tunneler-enabled --no-traversal 
+```
+
+</TabItem>
+<TabItem value="Public-Edge" label="Public Router with Edge" attributes={{className: styles.orange}}>
+
+```bash
+ziti edge create edge-router $ROUTER_NAME \
+                            --jwt-output-file $ROUTER_NAME.jwt
+```
+
+</TabItem>
+<TabItem value="Public-Fabric" label="Public Router" attributes={{className: styles.orange}}>
+To be added
```

Comment:
It seems best to comment this until we have something useful to publish.

Reply:
same as one before

Reply:
I will remove this for now, we will add the fabric router once the cli option to create transit routers is  added.